src/explorer/resultsList.ts

```diff
@@ -621,6 +624,31 @@ export class ResultsList {
         }
     }
 
+    /**
+     * Toggles the group row as well as any result rows that match the group
+     * @param row Group row to toggled
+     */
+    private expandGroupForResult(row: HTMLTableRowElement): void {
+
+        if (row.dataset.group === undefined) {
+            throw new Error("Expected to have collapse group.");
+        }
+
+        if (row.classList.contains(`${ToggleState.collapsed}`)) {
```

Comment:
Don't think the enum->string coercion is necessary.

Reply:
Agreed. Fixed here and elsewhere.